docs/repo-docs/getting-started/add-to-existing-repository.mdx

```diff
@@ -40,6 +40,10 @@ familiar with Turborepo.
 <Step>
 ### Install `turbo`
 
+<Callout type="good-to-know">
```

Comment:
Oh, good addition. Do you think you could move this into the tab for pnpm? Users of other package managers wouldn't need to see this information, so it would be great to localize. You won't need to have it be in a <Callout /> in there, as well.